readme.md

```diff
@@ -130,6 +130,17 @@ If you wish to be notified of new postings on your site you can use a plugin suc
 ### What language files are available? ###
 You can view (and contribute) translations via the [translate.wordpress.org](https://translate.wordpress.org/projects/wp-plugins/wp-job-manager).
 
+## Unit Testing ##
+
+The plugin contains all the files needed for running tests.
+Developers who would like to run the existing tests or add their tests to the test suite and execute them will have to follow these steps:
+1. `cd` into the plugin directory.
+2. Run the install script(you will need to have `wget` installed) - `bash bin/install-wp-tests.sh <db-name> <db-user> <db-pass> <db-host> <wp-version>`.
```

Comment:
I'm not sure I follow.  I haven't made any changes to the Gruntfile.

Reply:
In order for the unit tests to pass, external dependencies need to be installed. It might be worth adding something for setting up the dependencies in these instructions. Those commands are needed (with composer and npm installed) to set it up. You can ignore the bit about changing the Gruntfile.

Reply:
I'll add it into a new section later on and then reference it in these instructions.